amundsen_application/static/js/components/common/EntityCard/EntityCardSection/styles.scss

```diff
@@ -19,42 +20,45 @@
     margin-top: -4px;
     margin-left: 4px;
   }
-}
 
 
-button.edit-button {
-  height: 24px;
-  width: 24px;
-  border: none;
-  border-radius: 5px;
-  background-color: $gray-light;
-  -webkit-mask-image: url('/static/images/icons/Edit.svg');
-  mask-image: url('/static/images/icons/Edit.svg');
-}
-button.edit-button:hover {
-  background-color: $brand-color-4;
-}
-button.active-edit-button {
-  height: 24px;
-  width: 24px;
-  border: none;
-  border-radius: 5px;
-  background-color: $brand-color-1;
-  background-image: url('/static/images/icons/Edit_Inverted.svg');
-  -moz-box-shadow: inset 1px 1px 2px $brand-color-4;
-  -webkit-box-shadow: inset 1px 1px 2px $brand-color-4;
-  box-shadow: inset 1px 1px 2px $brand-color-4;
-}
-button.edit-button:focus,
-button.active-edit-button:focus {
-  outline: none;
-  box-shadow: 0 0 1pt 1pt $brand-color-4
-}
+  button.edit-button {
```

Comment:
I nested these styles under `.entity-card-section` because they were overlapping with the new edit buttons. Eventually we should be able to delete this.